### PR TITLE
Redfish api registries

### PR DESCRIPTION
**Pull request by uppalk1**
@jlongever @zyoung51 @amymullins @tannoa2 @keedya

### CODE REVIEW

**Comment by tannoa2** [comment]
test this please

File: lib/api/redfish-1.0/registry.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+var listRegistry = controller(function(req, res) {
+    var options = redfish.makeOptions(req, res);
+    options.files = [{
+      id: 'Base.1.0.0'
+    }];
+    return redfish.render('redfish.1.0.0.messageregistryfilecollection.1.0.0.json',
+                          'MessageRegistryFileCollection.json#/definitions/' +
+                          'MessageRegistryFileCollection',
+                          options);
```
Comment (zyoung51) [inline lib/api/redfish-1.0/registry.js]: Can you add the .catch() error handling to here and getRegistryFile() below?

**Comment by zyoung51** [comment]
+1 after the 2 error handlers are added

**Comment by tannoa2** [comment]
+1

**Comment by jlongever** [comment]
+1

**Pull request merged**